tx_test.go

```diff
@@ -602,6 +602,108 @@ func TestTx_CopyFile_Error_Normal(t *testing.T) {
 	}
 }
 
+// Ensure that a update that releases pages in ranges between read transactions
```

Comment:
To follow go-style function comment, do `TestTx_releaseRange ensures that...`

Reply:
Also I think the test comment should say what we mean by "properly".  In particular, this is a **correctness** test that verifies we don't "overfree".  (Unless I read the code wrong.)